.evergreen/functions.yml

```diff
@@ -452,14 +453,14 @@ functions:
           <<: *compass-env
         script: |
           eval $(.evergreen/print-compass-env.sh)
-          .evergreen/publish-dev-release.sh LATEST
+          .evergreen/publish-dev-release.sh release.json
```

Comment:
nit: i noticed this while looking for artifacts yesterday, but both the script name "publish-dev-release" and the task name publish-dev-release do not actually publish the dev release, which is kinda unexpected for someone who wasn't too involved.

How about we rename the script as "build-dev-release-info" and the task `dev-release-info` or something?

Reply:
made the change in [e0b94e6](https://github.com/mongodb-js/compass/pull/5868/commits/e0b94e64cdbaba312aeb0a59eb1d8b4c9e0fcc92)